src/mock-doc/test/element.spec.ts

```diff
@@ -264,7 +264,7 @@ describe('element', () => {
       expect(root.contains(root)).toEqual(true);
     });
 
-    it('returns false when a node is not the given node itself or not a descendant of the given node ', () => {
+    it('returns false when a node is not the given node itself or not a descendant of the given node', () => {
```

Comment:
one of the jest default rules is [`"valid-title"`](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/valid-title.md) which enforces, among other things, that titles not have any trailing or leading whitespace.